examples/cargo/BUILD

```diff
@@ -0,0 +1,28 @@
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_binary",
+    "rust_library",
+    "rust_test",
+)
+load(
+    "@io_bazel_rules_rust//cargo:cargo_build_script.bzl",
+    "cargo_build_script",
+)
+
+cargo_build_script(
+    name = "build_script",
+    srcs = ["build_script.rs"],
+    data = ["test.txt"],
+    crate_features = ["bleh"],
+)
+
+rust_library(
+    name = "lib",
+    srcs = ["lib.rs"],
+    deps = [":build_script"],
+)
```

Comment:
(No action Item)

I like this format - succinct and clear. Thanks for the updates!

Reply:
Thanks :)